modules/cdn/profile/main.bicep

```diff
@@ -9,23 +9,19 @@ param name string
 param location string = resourceGroup().location
 
 @allowed([
-  'Standard_Verizon'
+  'Custom_Verizon'
+  'Premium_AzureFrontDoor'
+  'Premium_Verizon'
+  'StandardPlus_955BandWidth_ChinaCdn'
+  'StandardPlus_AvgBandWidth_ChinaCdn'
+  'StandardPlus_ChinaCdn'
+  'Standard_955BandWidth_ChinaCdn'
   'Standard_Akamai'
+  'Standard_AvgBandWidth_ChinaCdn'
+  'Standard_AzureFrontDoor'
   'Standard_ChinaCdn'
   'Standard_Microsoft'
-  'Premium_Verizon'
-  'Premium_Akamai'
-  'Premium_ChinaCdn'
-  'Premium_Microsoft'
-  'Custom_Verizon'
-  'Custom_Akamai'
```

Comment:
I see some were removed. Are they 'not supported'?

Reply:
They are not suported. This is the supoprted list.